src/js/vic-v2/containers/ConfirmationPage.jsx

```diff
@@ -34,21 +37,39 @@ class ConfirmationPage extends React.Component {
     const response = form.submission.response
       ? form.submission.response.attributes
       : {};
+    const { veteranFullName, verified } = form.data;
+    const submittedAt = moment(form.submission.submittedAt);
 
     return (
       <div>
-        <h3 className="confirmation-page-title">Claim received</h3>
-        <p>We process claims in the order we receive them.</p>
-        <p>
-          We may contact you for more information or documents.<br/>
-          <i>Please print this page for your records.</i>
-        </p>
+        <p>You have completed your application.<br/>
+        We process applications and print cards in the order we receive them.</p>
+
+        <h4>What happens after I apply?</h4>
```

Comment:
using h4 here results in the following aXe violation:
```
Heading levels should only increase by one
 Inspect Node Highlight

1 of 1

Issue description
Ensures the order of headings is semantically correct

Impact: moderate
 Learn more
Element location
#react-root > div > .row > div > div > div:nth-child(3) > h4
Element source
<h4>What happens after I apply?</h4>
To solve this violation, you need to:
Fix the following:
Heading order invalid
```

Reply:
I'm not getting this error when I run axe in the browser. How are you generating it?

Reply:
I used aXe Coconut.